docs/_data/shapes.csv

```diff
@@ -0,0 +1,203 @@
+  </svg>"
+Trapezoid,"A four-sided shape with one pair of parallel sides, used to indicate hierarchical relationships or steps in a process.","- Representing funnels in process flows (e.g., sales funnels)
+- Indicating converging or diverging processes
+- Showing hierarchical structures with varying levels of detail",Dependency: Often used to show narrowing or widening of scope in processes,,"<svg viewBox=""0 0 40 40"" fill=""none"" xmlns=""http://www.w3.org/2000/svg"">
+<path d=""M8.5 3H32.5L39.5 37H1.5L8.5 3Z"" fill=""#00B39F"" fill-opacity=""0.1"" stroke=""#00B39F"" stroke-miterlimit=""10"" stroke-width=""1""/>
+</svg>"
+SqlJob,,,,,"<svg viewBox=""0 0 40 40"" fill=""none"" xmlns=""http://www.w3.org/2000/svg"">
+<circle cx=""20"" cy=""20"" r=""19"" fill=""#00B39F"" fill-opacity=""0.1"" stroke=""#00B39F"" stroke-width=""2""/>
+<text x=""50%"" y=""50%"" dominant-baseline=""middle"" text-anchor=""middle"" font-family=""Arial, sans-serif"" font-size=""10"" fill=""#00B39F"">SQL</text>
+</svg>"
+User,"This node represents an individual user, often depicted with an icon or simple human silhouette.","- Representing end-users or user roles in system diagrams
```

Comment:
Comment out this shape given that it isn't included in releases yet, @sahu-01.

// @aabidsofi19